web-console/4.0-designs/role-based-access-control/role-based-access-control.md

```diff
@@ -0,0 +1,34 @@
+# Role Based Access Control (RBAC)
+
+## Object Lists
+
+![no access to a list in a single projects](img/list-single.png)
```

Comment:
I don't know if the inline notifications belong at the bottom of the empty state message. Feels like this would be the first thing, but hard to tell without actual content in there maybe. I think they also typically span the page width and are not centered. These are probably all questions for visuals though!

Reply:
yeah, I guess I intended that to be more of a code snippet than a notification really. I can update the visuals and maybe it'll make more sense